### PR TITLE
Add fs info final

### PR DESCRIPTION
**Pull request by agrover**
Final-review PR for #180.

### CODE REVIEW

**Comment by agrover** [comment]
Let's get this in please :smiley:

File: src/stratis_cli/_actions/_data.py [hunk 1]
```diff
@@ -131,15 +131,24 @@
...
 </method>
+<property name="Created" type="s" access="read">
+<annotation name="org.freedesktop.DBus.Property.EmitsChangedSignal" value="const"/>
+</property>
 <property name="Devnode" type="s" access="read">
 <annotation name="org.freedesktop.DBus.Property.EmitsChangedSignal" value="const"/>
 </property>
+<property name="MountPoints" type="as" access="read">
+<annotation name="org.freedesktop.DBus.Property.EmitsChangedSignal" value="false"/>
+</property>
```
Comment (tasleson) [inline src/stratis_cli/_actions/_data.py]: Previous PR stated you were removing this property?

**Comment by agrover** [comment]
final -> work PR, since MountPoints API is now removed.

**Comment by agrover** [comment]
#185 is final2 version of this.

**Pull request closed without merging**